egs/csj/s5/local/csj_train_lms.sh

```diff
@@ -42,6 +42,7 @@ fi
 text=$1    
 lexicon=$2 
 dir=$3     
+. path.sh
```

Comment:
do `. ./path.sh`
(in case there is a path.sh in a directory on the current PATH, or . is not on the path).
no need to squash
I'll merge to-morrow